spaces/boostedFinance/index.json

```
@@ -0,0 +1,11 @@
+{
+  "key": "boostedFinance",
```

Key should not contain uppercase 
```suggestion
  "key": "boosted-finance",
```